src/content_manager.cc

```diff
@@ -1,1768 +1,1811 @@
-/*MT*
```

Comment:
Why the permission change? We just end up with a giant diff... does it have changes too?

Reply:
Happened by accident. Sometimes my Samba share gets confused :-)

Reply:
Thanks for looking!